clients/common/hipsolver_datatype2string.cpp

```diff
@@ -88,3 +100,19 @@ hipsolverFillMode_t char2hipsolver_fill(char value)
     }
     return HIPSOLVER_FILL_MODE_LOWER;
 }
+
+hipsolverSideMode_t char2hipsolver_side(char value)
+{
+    switch(value)
+    {
+    case 'L':
+        return HIPSOLVER_SIDE_LEFT;
+    case 'R':
+        return HIPSOLVER_SIDE_RIGHT;
+    case 'l':
+        return HIPSOLVER_SIDE_LEFT;
+    case 'r':
+        return HIPSOLVER_SIDE_RIGHT;
+    }
+    return HIPSOLVER_SIDE_LEFT;
```

Comment:
If there are two identical cases, I would use fallthrough. e.g.
```
    case 'l':
    case 'L':
        return HIPSOLVER_SIDE_LEFT;
    case 'r':
    case 'R':
        return HIPSOLVER_SIDE_RIGHT;
    }
```
The final `return HIPSOLVER_SIDE_LEFT;` concerns me a little. It appears to be a case that should never happen. It would be nice to have some warning if it ever does, either via an assertion or an exception.